### ISSUE TITLE
trailing comma with --android

### ISSUE THREAD

**Issue by mtrakal** [nbadal/ktlint-intellij-plugin#332]
Plugin ignores settings in `.editorconfig` when enabled `--android` mode but used trailing comma:
```
trailing-comma-on-call-site = true
ij_kotlin_allow_trailing_comma = true
```

Plugin mark all trailing comma as `Unnecessary trailing comma before ")" (trailing-comma-on-call-site)` but it's valid for our project if it's defined in `.editorconfig` file.

**Current state:**
show warning, for android project with trailing commas.

**Expected state:**
don't ignore settings in EditorConfig file with `--android` mode enabled in Plugin settings.

**Comment by kenyee** [comment]
this might be fixed when it's updated to use ktlint 1.0.0 underneath... --android no longer seems to be an option w/ 1.0.0

**Comment by nbadal** [comment]
Working on 1.0.0 support now, and yes that should come with the new editorconfig code style changes 🤞

**Comment by kenyee** [comment]
I don't think bumping kotlinter to the latest will grab ktlint 1.0.0...looks like kotlinter only supports ktlint 0.50.x.

Not sure if doing the resolutionStrategy forcing they recommend will work too...it has a slightly different API.

**Comment by paul-dingemans** [comment]
> this might be fixed when it's updated to use ktlint 1.0.0 underneath... --android no longer seems to be an option w/ 1.0.0

Instead of `--android` you have to set `.editorconfig` property `ktlint_code_style=android_studio`.
